### ISSUE TITLE
Could you add my BBS to vd-view

### ISSUE THREAD

**Issue by brianmartin2001** [simonlaszcz/vd-view#5]
Hi Simon ,

I've recently created a BBS which runs on my BBC Master computer.
It's a Viewdata system and has nearly 300 pages of ART and info in there.
It is accessible by TCP/IP Dialup and through a JS link on the web that the guys at
CCL4 provided.

VD-VIEW is more sophisticated and better for web browsing so I wondered if there was a possibility
you could provide a link to my BBS through VD-VIEW.

It's called The Night Owl BBS and at the moment it is not up at regular times
but that will change once I get messaging on the Host sorted. For now it is 
sort of read only , although there is a Sysop chat function.

nightowlbbs.ddns.net:6400

further details and some sample  pages are visible on this thread here below and access details

thanks 
Brian

[the Night Owl BBS - stardot.org.uk](https://stardot.org.uk/forums/viewtopic.php?t=274

**Comment by simonlaszcz** [comment]
Yes of course. I'd be happy to. I'll add it sometime over the coming week 
- Simon

**Comment by brianmartin2001** [comment]
Many thanks Simon , I think it will bring me some more traffic. It's a really nice display. 
CheersBrian
    On Tuesday, 23 January 2024 at 11:28:45 GMT, Simon Laszcz ***@***.***> wrote:  
 
 


Yes of course. I'd be happy to. I'll add it sometime over the coming week
   
   - Simon

**Comment by simonlaszcz** [comment]
Hi Brian.

Many thanks.

I tried connecting this evening. Opened a socket (well the code did ;-)) and attempted to read but nothing was received. Do you expect any kind or preamble/handshaking or does your BBS just start sending as soon as it gets a new connection?

- Simon

**Comment by brianmartin2001** [comment]
It's only a part time BBS Simon. It wasn't running tonight. It will send as soon as carrier is established I think. I could set it up now if you want ? Or try tomorrow evening ?
Thanks againBrian
    On Wednesday, 24 January 2024 at 21:48:30 GMT, Simon Laszcz ***@***.***> wrote:  
 
 


Hi Brian.

Many thanks.

I tried connecting this evening. Opened a socket (well the code did ;-)) and attempted to read but nothing was received. Do you expect any kind or preamble/handshaking or does your BBS just start sending as soon as it gets a new connection?
   
   - Simon

**Comment by simonlaszcz** [comment]
Ah OK, no probs.  Message me next time it's running.

- Simon

**Comment by brianmartin2001** [comment]
I will set it up this evening for you about 7 pm onwards. Send me a message if you have any issues .. I can meet you on the BBS if you get connected..
Thank you
Brian
    On Thursday, 25 January 2024 at 09:33:42 GMT, Simon Laszcz ***@***.***> wrote:  
 
 


Ah OK, no probs. Message me next time it's running.
   
   - Simon

**Comment by brianmartin2001** [comment]
It's up now Simon if that suits.

nightowlbbs.ddns.net:6400
2400 baud 8E1
    On Thursday, 25 January 2024 at 09:33:42 GMT, Simon Laszcz ***@***.***> wrote:  
 
 


Ah OK, no probs. Message me next time it's running.
   
   - Simon

**Comment by simonlaszcz** [comment]
Thanks Brian. 
It connected OK so all looks good. That's a nice looking frame! 
I don't have much time tonight but I'll deploy it ASAP then register and go exploring.
- Simon

**Comment by brianmartin2001** [comment]
Can I try it out myself now on the browser yet  Simon ?
There should be quite a bit if interest in this if get the word out I think .
ThanksVrian
    On Thursday, 25 January 2024 at 19:46:47 GMT, Simon Laszcz ***@***.***> wrote:  
 
 


Thanks Brian.
It connected OK so all looks good. That's a nice looking frame!
I don't have much time tonight but I'll deploy it ASAP then register and go exploring.
   
   - Simon

**Comment by brianmartin2001** [comment]
I just tried there. Dont see it on the choices tab with NXTeL , CCL4 , Telstar etc
    On Thursday, 25 January 2024 at 19:46:47 GMT, Simon Laszcz ***@***.***> wrote:  
 
 


Thanks Brian.
It connected OK so all looks good. That's a nice looking frame!
I don't have much time tonight but I'll deploy it ASAP then register and go exploring.
   
   - Simon

**Comment by simonlaszcz** [comment]
Hi Brian,
I was just trying it locally before.
I was having a few Azure deployment problems, but it's now deployed OK
-- Simon

**Comment by brianmartin2001** [comment]
That is superb Simon , thanks so much for doing this. It's a great viewer, I really like it.  Full screen ability makes it feel just like a viewdata terminal. 
Again , many thanks
Brian
    On Friday, 26 January 2024 at 10:08:49 GMT, Simon Laszcz ***@***.***> wrote:  
 
 


Hi Brian,
I was just trying it locally before.
I was having a few Azure deployment problems, but it's now deployed OK
-- Simon

**Comment by simonlaszcz** [comment]
Thanks Brian - much appreciated.

**Comment by brianmartin2001** [comment]
Hi Simon , used it last night as did another user. Really great. Any reason why the double height is a little corrupted though. Just wondered if it's fixable or something I should avoid using on my board so there's less impact for folks when using vd-view
CheersBrian
    On Friday, 26 January 2024 at 10:35:30 GMT, Simon Laszcz ***@***.***> wrote:  
 
 


Thanks Brian - much appreciated.

**Comment by simonlaszcz** [comment]
Hi Brian.
Double height works elsewhere but it's not immune to bugs as I've fixed a few related to DH in the past.
Could you send me an example frame? In vd-view can you:
1. Navigate to the frame
2. Save the frame - click the disk icon in the control panel
3. Go to the frame in the 'saved frames' section
4. Click copy
That will give you a base 64 string that you can save to file and post here.
- Simon

**Comment by simonlaszcz** [comment]
Or you can send me a binary file straight from your BBS if that's easier

(sorry - wasn't sure you'd get notified of that last edit)

**Comment by brianmartin2001** [comment]
I ll put the host up tonight, and login through vd view and send you.  
Thanks Simon
Brian
    On Saturday, 27 January 2024 at 20:41:12 GMT, Simon Laszcz ***@***.***> wrote:  
 
 


Hi Brian.
Double height works elsewhere but it's not immune to bugs as I've fixed a few related to DH in the past.
Could you send me an example frame? In vd-view can you:
   
   - Navigate to the frame
   - Save the frame - click the disk icon in the control panel
   - Go to the frame in the 'saved frames' section
   - Click copy   
That will give you a base 64 string that you can save to file and post here.
   
   - Simon

**Comment by simonlaszcz** [comment]
Thanks.
If you can, a screen shot of what it should look like would also be useful.
Cheers

**Comment by brianmartin2001** [comment]
Hi Simon ,
sorry late with this, unexpected issues on the BBS. Looks likeI have lost all the accounts, or something has been corrupted.
Anyway I accessed the example of double height , best is the Blog section.I see it has changed and a bit improved from where it was initially.Have you been working on it? The double height of the text is unaffectedbut the graphic around it are. When I saw it the other day the text was unreadable. 
there is a raw data file here and  a PNG I took of what should be the same frame.

thanksBrian
    On Sunday, 28 January 2024 at 12:21:12 GMT, Simon Laszcz ***@***.***> wrote:  
 
 

Thanks.
If you can, a screen shot of what it should look like would also be useful.
Cheers

**Comment by simonlaszcz** [comment]
Thanks Brian,
I'll take a look over the next few days.
No, I haven't touched the site's code. DH is used quite a bit on the other BBSs and renders OK AFAIK.
It's possible you've found a bug though so I'll take a look, but I wonder what's changed 'your end' since the rendering has changed?

-Simon

**Comment by simonlaszcz** [comment]
Hi Brian,
I can't find the files. Maybe they get removed from email. Could you paste them directly into a GitHub comment?
Thanks

**Comment by brianmartin2001** [comment]
Hi Simon ,
Do you mean there are no attachments at all ?Or you can't open them? 
I have a GitHub account but hardly use it will try and do this at some point but the double height text is much better now
One other prob.On the login screen I noticed the cursor doesn't align with the input prompts for surname and password , although for first name it is ok. Just goes awry with the other entries . 
Have you set up an account yourself as yet? .Let me know if you want explore and I can log it up for you this evening.  You might get a better idea of any issues that way. 
CheersBrian


    On Monday, 29 January 2024 at 00:10:56 GMT, Simon Laszcz ***@***.***> wrote:  
 
 


Hi Brian,
I can't find the files. Maybe they get removed from email. Could you paste them directly into a GitHub comment?
Thanks

**Comment by simonlaszcz** [comment]
Hi Brian,

There are no attachments at all.

Regarding cursor alignment, perhaps you need to output another backspace. It's fine everywhere else (i.e. Telstar, CCL4)

OK. I'll login when I can

Simon

**Comment by brianmartin2001** [comment]
I was going to bring up the Host tonight for you Simon but I ve been running unexpectedl errands all night and I am only just in.
Tomorrow would be better. Let me know if it suits you 
Thanks for your help again.
Brian 
    On Tuesday, 30 January 2024 at 10:47:59 GMT, Simon Laszcz ***@***.***> wrote:  
 
 

Hi Brian,

There are no attachments at all.

Regarding cursor alignment, perhaps you need to output another backspace. It's fine everywhere else (i.e. Telstar, CCL4)

OK. I'll login when I can

Simon

**Comment by brianmartin2001** [comment]
I keep trying to mail you Simon but keeps bouncing the message.
    On Tuesday, 30 January 2024 at 10:47:59 GMT, Simon Laszcz ***@***.***> wrote:  
 
 

Hi Brian,

There are no attachments at all.

Regarding cursor alignment, perhaps you need to output another backspace. It's fine everywhere else (i.e. Telstar, CCL4)

OK. I'll login when I can

Simon

**Comment by simonlaszcz** [comment]
Hi Brian,
I've just been looking into the cursor positioning on the login frame. I traced it something like follows:

First name rendered
then "Please enter last name (** to clear)  " rendered
RS (back to origin)
16 * LF
16 * horizontal tab
4 * backspace
LF
10 * space <-- this overwrites the surname label and cursor will be in wrong place

**Comment by brianmartin2001** [comment]
Yeah I saw you had visited Simon, thanks.
So is that issue now fixed? It was only a minor thing but great if you have done it.

Cheer's Brian 
    On Friday 31 May 2024 at 20:59:08 BST, Simon Laszcz ***@***.***> wrote:  
 
 


Hi Brian,
I've just been looking into the cursor positioning on the login frame. I traced it something like follows:

First name rendered
then "Please enter last name (** to clear) " rendered
RS (back to origin)
16 * LF
16 * horizontal tab
4 * backspace
LF
10 * space <-- this overwrites the surname label and cursor will be in wrong place

**Comment by simonlaszcz** [comment]
Hi Brian. The LF issue is fixed now. It was a bit of a 'doh' moment when I realised it was clearly my code at fault

**Comment by brianmartin2001** [comment]
Thanks for sorting it Simon and looking at the BBS too.
I wanted to look at your fix  myself but still having trouble getting the vd view page to load up. 
It was fine on Thursday evening, see photos if Night Owl screen here , but seems not to load now.
Ant thoughts?
Thanks Brian
    On Saturday 1 June 2024 at 20:40:55 BST, Simon Laszcz ***@***.***> wrote:  
 
 


Hi Brian. The LF issue is fixed now. It was a bit of a 'doh' moment when I realised it was clearly my code at fault

**Comment by simonlaszcz** [comment]
Yes it does seem a bit laggy. I'll take a look. I did 'tweak' a couple of things

**Comment by brianmartin2001** [comment]
https://www.stardot.org.uk/forums/viewtopic.php?p=427122#p427122

All was fine Thurs
    On Saturday 1 June 2024 at 22:20:44 BST, Brian Martin ***@***.***> wrote:  
 
  Thanks for sorting it Simon and looking at the BBS too.
I wanted to look at your fix  myself but still having trouble getting the vd view page to load up. 
It was fine on Thursday evening, see photos if Night Owl screen here , but seems not to load now.
Ant thoughts?
Thanks Brian
    On Saturday 1 June 2024 at 20:40:55 BST, Simon Laszcz ***@***.***> wrote:  
 
 


Hi Brian. The LF issue is fixed now. It was a bit of a 'doh' moment when I realised it was clearly my code at fault

**Comment by brianmartin2001** [comment]
Just saw you login there, cheers for your help Simon, much appreciated 
    On Saturday 1 June 2024 at 22:41:47 BST, Simon Laszcz ***@***.***> wrote:  
 
 


Yes it does seem a bit laggy. I'll take a look. I did 'tweak' a couple of things

**Comment by simonlaszcz** [comment]
Seems to be taking a while for your Beeb to respond to each character. Maybe it needs a restart?

**Comment by brianmartin2001** [comment]
Sorry just read this Simon ,
Will log it back up in the morning or whenever if needed. Just let me know.
    On Saturday 1 June 2024 at 22:55:31 BST, Simon Laszcz ***@***.***> wrote:  
 
 


Seems to be taking a while for your Beeb to respond to each character. Maybe it needs a restart?

**Comment by simonlaszcz** [comment]
All seems to be working so I'll close this.

Very nice BBS by the way. Looks to be very popular

**Comment by brianmartin2001** [comment]
Thanks for sorting this Simon , and great to have you visit. Please come back and leave some messages about anything. Would be good to have someone of you knowledge contribute in the chat now and then.
Thanks very much
Brian
    On Monday 10 June 2024 at 21:25:54 BST, Simon Laszcz ***@***.***> wrote:  
 
 


All seems to be working so I'll close this.

Very nice BBS by the way. Looks to be very popular

**Comment by simonlaszcz** [comment]
I definitely will. I'm trying to fix a cursor positioning problem in vidtex (unix client) so you'll see me sign in a few times

**Comment by brianmartin2001** [comment]
Hi Simon , vd- view seems to be off line ... Any issues at the moment
Thanks Brian 
    On Tuesday 11 June 2024 at 16:24:54 BST, Brian Martin ***@***.***> wrote:  
 
  Thanks for sorting this Simon , and great to have you visit. Please come back and leave some messages about anything. Would be good to have someone of you knowledge contribute in the chat now and then.
Thanks very much
Brian
    On Monday 10 June 2024 at 21:25:54 BST, Simon Laszcz ***@***.***> wrote:  
 
 


All seems to be working so I'll close this.

Very nice BBS by the way. Looks to be very popular

**Comment by simonlaszcz** [comment]
Hi Brian.
As daft as it sounds I took it down due to the Online Safety Act, specifically Ofcom now enforcing it. It vaguely looks like the site is in scope and since I have no controls in place (and can't be bothered to) I decided to take it down. Once I've had time to look into it further I might bring it back online.
- Simon

**Comment by brianmartin2001** [comment]
Had no idea that would be something you d have to deal with.. crikey , police state.
Thanks for the update Simon 
    On Wednesday 19 March 2025 at 09:59:08 GMT, Simon Laszcz ***@***.***> wrote:  
 
  
Hi Brian.
As daft as it sounds I took it down due to the Online Safety Act, specifically Ofcom now enforcing it. It vaguely looks like the site is in scope and since I have no controls in place (and can't be bothered to) I decided to take it down. Once I've had time to look into it further I might bring it back online.
   
   - Simon

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***>
simonlaszcz left a comment (simonlaszcz/vd-view#5)
Hi Brian.
As daft as it sounds I took it down due to the Online Safety Act, specifically Ofcom now enforcing it. It vaguely looks like the site is in scope and since I have no controls in place (and can't be bothered to) I decided to take it down. Once I've had time to look into it further I might bring it back online.
   
   - Simon
